CHANGELOG.md

```diff
@@ -3,6 +3,74 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.13.0
+
+_Sep 7, 2023_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Fix `anchorRef` behavior on range pickers (#10077) @LukasTy
```

Comment:
Decided to propose a minor release because of this.
It could even be a breaking change in someone's book, based on their usage and perspective. 🤔